apps/coscrad-frontend/src/store/slices/resources/spatial-features/spatial-feature-slice.ts

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { buildInitialLoadableState } from '../../utils';
+import { buildReducersForThunk } from '../../utils/buildReducersForThunk';
+import { SPATIAL_FEATURE } from './constants';
+import { fetchSpatialFeatures } from './thunks';
+import { SpatialFeatureIndexState, SpatialFeatureSliceState } from './types';
+
+const initialState: SpatialFeatureSliceState =
```

Comment:
I wonder if the generic and the type on the const are both necessary. It seems redundant as the generic is essential a cast, while if we leave the type off the const declaration it should be inferred. Investigate further.

Reply:
I removed the type on the const declaration and the inference is fine.